src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -175,6 +175,13 @@ public void UpdateWith(JobOption? jobOption)
         return new DynamicJobFactory(scopedServiceProvider, Delegate);
     }
 
+    public bool IsUnnamedOrUnscheduledOrParameterlessTypedJob =>
```

Comment:
Given the description - is the check against `CronExpression is not null` important?

Reply:
Unscheduled was a shortcut for
- Isn't Cron based
- Doesn't run at Startup 

Without it 5 tests fail

![image](https://github.com/user-attachments/assets/9aac8da6-f7af-4e48-854b-124e6cf82af0)